solidity/core/contracts/Outbox.sol

```diff
@@ -166,7 +166,7 @@ contract Outbox is IOutbox, Version0, MerkleTreeManager, Common {
         override
         returns (bool)
     {
-        // Checkpoint indices are one-indexed.
+        // Checkpoints are zero-indexed, but checkpoints of index 0 are disallowed
```

Comment:
Wondering something like this for the mapping definition in `Common.sol` would be helpful for future readers.

I mention this because usually first thing I do when reading from a contract's public storage is to look at the definition and any notes around it